--- a/node_modules/@react-native-menu/menu/ios/MenuViewManager.mm
+++ b/node_modules/@react-native-menu/menu/ios/MenuViewManager.mm
@@ -1,13 +1,13 @@
 #import <React/RCTViewManager.h>
 #import <React/RCTUIManager.h>
-#import "RCTBridge.h"
+#import <React/RCTBridge.h>
 
 #ifdef RCT_NEW_ARCH_ENABLED
 // NEW ARCH
 #import "MenuView.h"
 #else
 // OLD ARCH
-#import <react_native_menu/react_native_menu-Swift.h>
+#import <react_native_menu-Swift.h>
 #endif
 
 @interface MenuViewManager : RCTViewManager
