diff --git a/node_modules/expo-appcenter/plugin/build/ios/appDelegate.js b/node_modules/expo-appcenter/plugin/build/ios/appDelegate.js
index 0c34098..16a0cfd 100644
--- a/node_modules/expo-appcenter/plugin/build/ios/appDelegate.js
+++ b/node_modules/expo-appcenter/plugin/build/ios/appDelegate.js
@@ -19,9 +19,9 @@ function modifyObjcAppDelegate(contents) {
     }
     // Add invocation
     if (!contents.includes(methodInvocationBlock)) {
-        contents = contents.replace(/\[super application\:application didFinishLaunchingWithOptions\:launchOptions\]/g, `${methodInvocationBlock}
+        contents = contents.replace(/return \[super application\:application didFinishLaunchingWithOptions\:launchOptions\]/g, `${methodInvocationBlock}
       
-[super application:application didFinishLaunchingWithOptions:launchOptions]`);
+return [super application:application didFinishLaunchingWithOptions:launchOptions]`);
     }
     return contents;
 }
